arthas-api/index.js

```diff
@@ -30,7 +30,8 @@ const PORT = 8000;
  * Config
  */
 
-const numCPUs = availableParallelism();
+// const numCPUs = availableParallelism();
+const numCPUs = 1;
```

Comment:
This approach works in a single instance, but would still need another solution to share the stateful `chat` method across multiple.